keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -199,6 +254,25 @@ CPU and memory limit configurations from runtime.
 These CRI changes are a separate effort that does not affect the design
 proposed in this KEP.
 
+### Risks and Mitigations
+
+1. Backward compatibility: When Pod.Spec.Containers[i].Resources becomes
+   representative of desired state, and Pod's true resource allocations are
+   tracked in Pod.Status.ContainerStatuses[i].ResourcesAllocated, applications
+   that query PodSpec and rely on Resources in PodSpec to determine resource
+   allocations will see values that may not represent actual allocations. As a
+   mitigation, this change needs to be documented and highlighted in the
+   release notes, and in top-level Kubernetes documents.
+1. Resizing memory lower: Lowering cgroup memory limits may not work as pages
```

Comment:
memory qos may help on v2 depending on a threshold we apply for `memory.high`.

see: https://github.com/kubernetes/enhancements/pull/2571